### PR TITLE
Fix #7012: Slider add min max range functionality

### LINKED ISSUE #7012: Slider: min max range styles missing
image
Jquery UI Slider have 4 range mode and we are supporting only 2 of them. Also I think we need "min" range by default.

### PR DESCRIPTION
**Pull request by buunsal**
image
I also commited new demos to showcase. Basic one not acceptable and contrary to what we use in our other ui libraries.
Primeng sample: [Primeng Slider](https://primefaces.org/primeng/showcase/#/slider)

But i need code review.  It could be written better maybe.

### CODE REVIEW

File: src/main/java/org/primefaces/component/slider/SliderRenderer.java [hunk 1]
```diff
@@ -106,13 +106,24 @@ protected void encodeScript(FacesContext context, Slider slider) throws IOExcept
...
+        }
+
+        else if ("false".equals(range)) {
+            wb.attr("range", false);
+        }
+
+        else {
+            wb.attr("range", range);
+        }
+
```
Comment (melloware) [inline src/main/java/org/primefaces/component/slider/SliderRenderer.java:124]: How about this instead:
```java
                switch (range) {
		case "true":
		case "false":
			 wb.attr("range", Boolean.valueOf(range));
			break;
		default:
			wb.attr("range", range);
			break;
		}
```

**Review by melloware — CHANGES_REQUESTED** [review]

File: src/main/java/org/primefaces/component/slider/SliderRenderer.java (hunk 1 above)
Comment (buunsal) [reply src/main/java/org/primefaces/component/slider/SliderRenderer.java:124]: Ok for me. I added your code.

**Comment by tandraschko** [comment]
you also may adjust the taglib xml

**Comment by melloware** [comment]
And \docs\migrationguide\10_0_0.md

**Comment by buunsal** [comment]
added

**Review by melloware — APPROVED** [review]

**Pull request merged**